sourcegraph/sourcegraph
# Quality local search

##### Reviewer tasks

*AUTHOR: Add other outstanding tasks for the reviewer. Remove irrelevant items below.*

- [ ] SECURITY: reviewer approves the security-related changes
- [ ] PERF: reviewer approves of the site perf impact of this change
- [ ] DOCS: reviewer approves the docstrings (or the justified omission thereof)
- [ ] UNIT-TEST: reviewer approves the unit tests (or the justified omission thereof)
- [ ] E2E-TEST: reviewer approves end-to-end tests (or the justified omission thereof)

##### Test plan

Test on staging:
- Go to gorilla/mux.
- Search for something locally with the syntax you're accustomed to be supported by global search. Observe that it works.
- Now go to a specific commit in gorilla/mux that has been built. Observe that search works here, too.
- Try to break the search box or make it run super slow with queries.
